packages/relay/src/lib/clients/cache/redisCache.ts

```diff
@@ -310,4 +310,18 @@ export class RedisCache implements IRedisCacheClient {
     this.logger.trace(`${requestIdPrefix} pushing ${serializedValue} to ${key} on ${callingMethod} call`);
     return result;
   }
+
+  /**
+   * Retrieves all keys matching a pattern.
+   * @param {string} pattern The pattern to match
+   * @param {string} callingMethod The name of the calling method
+   * @param {string} [requestIdPrefix] The optional request ID prefix
+   * @returns {Promise<string[]>} The list of keys matching the pattern
+   */
+  async keys(pattern: string, callingMethod: string, requestIdPrefix?: string): Promise<string[]> {
+    const client = await this.getConnectedClient();
+    const result = await client.keys(pattern);
+    this.logger.trace(`${requestIdPrefix} retrieving keys matching ${pattern} on ${callingMethod} call`);
```

Comment:
nit: for consistency, let's tryna match up the content of the log of these keys() methods. This line and this line https://github.com/hashgraph/hedera-json-rpc-relay/pull/2931/files#diff-aa7ffa04dc980cc6bdc951ea3c201ed60d6a390851503a559371a37d73bd3014R228 can be more similar.